Configuration/PyReleaseValidation/python/relval_nano.py

```diff
@@ -257,16 +264,18 @@ def subnext(self):
 workflows[_wfn()] = ['NANOdata130Xrun3', ['MuonEG2023MINIAOD13.0', 'NANO_data13.0_prompt', 'HRV_NANO_data']]
 workflows[_wfn()] = ['muDPGNANO130Xrun3', ['ZMuSkim2023DRAWRECO13.0', 'muDPGNANO_data13.0']]
 workflows[_wfn()] = ['muDPGNANOBkg130Xrun3', ['ZeroBias2023DRAW13.0', 'muDPGNANOBkg_data13.0']]
-workflows[_wfn()] = ['muPOGNANO_data13.0', ['MuonEG2023MINIAOD13.0', 'muPOGNANO_data13.0']]
+workflows[_wfn()] = ['muPOGNANO130Xrun3', ['MuonEG2023MINIAOD13.0', 'muPOGNANO_data13.0']]
 workflows[_wfn()] = ['EGMNANOdata130Xrun3', ['MuonEG2023MINIAOD13.0', 'EGMNano_data13.0']]
+workflows[_wfn()] = ['jmeNANOdata130Xrun3', ['MuonEG2023MINIAOD13.0', 'jmeNano_data13.0']]
 workflows[_wfn()] = ['lepTimeLifeNANO_data13.0', ['MuonEG2023MINIAOD13.0', 'lepTimeLifeNANO_data13.0']]
```

Comment:
@mbluj : can you please create a set of PR to follow the `NANOdata130Xrun3` and `NANOmc132X` suffix syntax ?

Reply:
Sure, but let me understand better: do you want me to add "lepTimeLifeNANO" test workflows to `NANOdata130Xrun3` and `NANOmc132X` workflow chains? Or you mean something else?

Reply:
`lepTimeLifeNANO_data13.0` --> `lepTimeLifeNANOdata130Xrun3`, etc